saleor/core/utils/editorjs.py

```diff
@@ -7,15 +7,16 @@
 HYPERLINK_TAG_WITH_URL_PATTERN = r"(.*?<a\s+href=\\?\")(\w+://\S+[^\\])(\\?\">)"
 
 
-def clean_editor_js(definitions: dict):
+def clean_editor_js(definitions: dict, to_string=False):
     """Sanitize a given EditorJS JSON definitions.
 
     Look for not allowed URLs, replaced them with `invalid` value, and clean valid ones.
     """
```

Comment:
Could you add a short description for what the flag`to_string` is used to? I also think that we should change `to_string` to keyword-only parameter to force passing this by key.  For me call like `clean_editor_js(some_data, True)` is not clear and I need to go to the function to check what the `True` value is.

```suggestion
def clean_editor_js(definitions: dict, *, to_string=False):
```

Reply:
Good point, I will do that